MetaMask/metamask-extension
# UX: Multichain: Display total account balance in account menu

## **Description**
For our multichain effort, we want to show total balances in both the home screen as well as the account menu and any other place we have `AccountListItem` components.  This PR creates a `useAccountTotalFiatBalance` hook that returns tokens and the total fiat balance.

## Fixes
https://github.com/MetaMask/MetaMask-planning/issues/1426

## **Manual testing steps**

1. Start the extension with `MULTICHAIN=1 yarn start`
2. Have an account with an ETH balance and token balances
3. On the home screen, ensure the large displayed balance is the sum of ETH and token fiat balances
4. Open the Accounts Menu
5. Ensure that each in the menu displays an fiat value that is the sum of the relevant account

With the current build, there should be no changes.

## **Screenshots/Recordings**

https://github.com/MetaMask/metamask-extension/assets/46655/87132195-98bc-4ad1-8414-b0999daf3037

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
